### ISSUE TITLE
Project: Paperless Hiring Process

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#428]
## Description
Optimize Austin Transportation's HR hiring processes by digitizing paperwork and reducing the need for manual data entry. 

### Resources
 - [Project request](https://docs.google.com/document/d/1ghuo7geHEpCPbCD6AlO-kptP2gXOP1ynWldj4fWf3uA/edit#)
 - [Drive](https://drive.google.com/drive/folders/1XBVT2-yv87knZIdv6Hf-uzH6ce_TcUip)

---
This Github issue represents a project of Austin Transportation's [Data & Technology Services](https://transportation.austintexas.io/about/) team. Project status is documented regularly in the comments below.

**Comment by johnclary** [comment]
From Adan;

> As I promised, in this email you will find a pseudo ‘interview >binder’ that we put together prior to an interview.
> 
The HR person going to the interview extracts from the following >sources:
>Cover – created in Excel
Interview Guide – from HRD/CitySpace (>http://cityspace.ci.austin.tx.us/departments/hrm/hrforms)
Minimum/Preferred Qualifications – Mins come from HRD/CitySpace & >Preferred comes from hiring manager
>Job Posting – eCareer
Job Description – COA website (http://austintexas.gov/hr/jobdesc/>job_title_pay.cfm)
>Application 1 – eCareer
>Cover Letter 1 – eCareer
>Resume 1 – eCareer
>Application 2 – eCareer
>Cover Letter 2 – eCareer
>Resume 2 – eCareer
>Application 3 – eCareer
>Cover Letter 3 – eCareer
>Resume 3 – eCareer
>Application 4…
> 
As of today, we are not looking to create forms for entry (notes/signatures/scorings), but Nathan would like for some discussion and planning about this at our meeting next week. I can provide copies of those prior, if that interests you. I was planning on bringing >those to the meeting since this would be a future/proposed project.
> 
Copying Amenity, because I’ve worked with her before. Did not >include her on the original discussion.
> 
Feel free to reach out to me for further discussion. The senior >Advisors can also add to what HR is aiming for.
>Best,
> 
>Adan Medina

**Comment by johnclary** [comment]
Corporate HR via Nathan B:

> I know several departments have developed things to make their HR work easier. As long as eCareer is still used as the ATS, and Banner is still kept up to date, if you are able to get information and make it more usable, that is fine.  Once we get the new HCM, the whole City will start to use it, and anything in the departments will need to move to the City solution.  One thing to keep in mind is that as we start the full HCM process and continue with KRONOS, many HRD and Controller’s office resources will be focused on those projects.  We will not be able to help with other department solutions.
>
> If it was a few years ago and you asked, I would have said it makes a lot of sense to move forward with an interim solution – the HCM Governing Board with CTM was hearing those items more frequently at that time.  Now, with my current Gartner engagement, and funding strategy, it seems to be moving more quickly and I hope to have a recommended HCM solution within the year.  After we award, it may be a while to get it up and running, but really, starting with nothing may be advantageous at this point to be able to get something stood up quicker.

**Comment by johnclary** [comment]
Adding [this signature page](https://app.zenhub.com/files/140626918/2d56b74f-d5c3-4dae-a296-62d15bc52898/download) document which was recently circulated to staff. It's being used to track employee receipt of open enrollment paperwork as well a t-shirt size. It's a twofer!

**Comment by TracyLinder** [comment]
@johnclary what are the next steps for this? Should we assign a PM for discovery & get that going, or stick it in the backlog?

**Comment by johnclary** [comment]
@TracyLinder backlog for now. thanks!

**Comment by johnclary** [comment]
Another known paper form: the Appeal for Accrued Leave Donation form.

**Comment by amenity** [comment]
Discuss HRD IT Projects with Perry P.

_John, Amenity, Perry_

**Active projects**

- Primary focus is KRONOS electronic timesheets, team full-time on that
  - Testing with pilot groups: HRD, CTM, Finance will be kicking off late February
  - Just finished first end-to-end testing and identified probklems, but Kronos released a new version so may need to start over
  - In parallel, putting together training material
  - Will then do Austin Energy and Fire; five different phases and subphases:
  - Transportation estimated date would be February 2021
- New Human Capital Management software
  - Applicant tracking (including paperless interview support) and onboarding, leave accrual, "do it all" but modules will be implemented in different tiers
    - First level is basic personnel
    - Second level, manager and employee self-service; maybe applicant tracking and onboarding 
    - Third: Nice to haves, e.g. LMS safety, drug and alcohol
  - TBD whether payroll office will be able to go with new system - maybe hyprid with KRONOS and Banner staying in place with new configuration and then new HCM running in parallel
  - Timeline:
    - RFP being put together now, goal is to have that out for 2 months
    - Will begin implementation with vendor over the summer
    - Already have funding, haven't in past so likely 
    - Begin with pilot departments and first tier modules - "vanilla departments" vs departments with different labor types

**Comment by amenity** [comment]
*Status update*
@dianamartin has been working with HR to [review potential tools and workflows ](https://github.com/cityofaustin/atd-data-tech/issues/1297). They have purchased Chromebooks for these activities and should be able to begin piloting solutions during interviews.

**Comment by johnclary** [comment]
Nathan reached out to us to discuss further HR process which they would like to move online. We're waiting for him to get back to us with a list of needs, and then we can huddle and meet with HR to discuss.

**Comment by dianamartin** [comment]
@johnclary Scott told me the Chrome books have been in for a couple months already

**Comment by johnclary** [comment]
super. then do we just need to get them to HR?

**Comment by moorerst** [comment]
Check in meeting with HR staff (Nathan & Meredith)
Pipeline projects they are also excited about:

- SSPR process for next fiscal year (DSD may have a process in place we can look at)
- Revised HR Sharepoint site
- New Hire paperwork
